tslint-eslint-config-5minds/README.md

```diff
@@ -0,0 +1,11 @@
+# TSLint + ESLint config for 5Minds projects
```

Comment:
`for 5Minds projects` klingt so, als wäre es nur für 5Minds Projekte gedacht. Da es aber ein Open Source Release ist, würde ich hier sowas wie `TSLint + ESLint rule set [recommendation] by 5Minds` vorschlagen.